Library/Formula/fsevent_watch.rb

```diff
@@ -0,0 +1,16 @@
+class FseventWatch < Formula
+  homepage "https://github.com/proger/fsevent_watch"
+  url "https://github.com/proger/fsevent_watch/archive/v0.1.tar.gz"
+  sha1 "abab1599a2143dd24bcb6a78e0bc29a30337119e"
+
+  head "https://github.com/proger/fsevent_watch.git"
+
+  def install
+    bin.mkpath
+    system "make", "install", "PREFIX=#{prefix}", "CFLAGS=-DCLI_VERSION=\\\"#{version}\\\""
+  end
+
+  test do
+    system "fsevent_watch", "-V"
```

Comment:
The `-V` option is actually the short version of `--version`. I'm modifying to `--version` for more clarity. It will be difficult to write a more elaborate test due to the nature of the product (it sits and listens for changes in a folder and must be stopped with an interrupt).